### ISSUE TITLE
Perseus exercise triage

### ISSUE THREAD

**Issue by mjptak** [learningequality/ka-lite#3082]
branch: develop
steps to reproduce: run exercise and to go to hint  

This exercise works but the hint takes you to a video url which targets the Khan Academy mothership.  
http://127.0.0.1:8008/learn/khan/humanities/renaissance-reformation/early-renaissance1/sculpture-architecture-florence/brunelleschi-and-ghiberti-s-sacrifice-of-isaac-quiz/  

This appears to affect all of the assessment items used in this exercise..

**Comment by mjptak** [comment]
It turns out that 878 assessment items reference a video (searched for video in the assessment_item_hints).  I presume that some sort of blanket change can be made to redirect to the local KA Lite video in these cases.

**Comment by jamalex** [comment]
Thanks, @mjptak! This is really helpful.

We should get these cleaned up prior to release.

**Comment by jamalex** [comment]
Interestingly, the quiz mentioned above links to a KA video that no longer exists, so it's broken on their site as well:
https://www.khanacademy.org/humanities/renaissance-reformation/early-renaissance1/sculpture-architecture-florence/e/brunelleschi-and-ghiberti-s-sacrifice-of-isaac-quiz

But in general, we should be able to do a lookup and replacement.

**Comment by jamalex** [comment]
Oh, looks like they're just using the incorrect field (the "id", with multiple dashes in a row, rather than the "slug") in generating their video link. So, in the process of fixing up the URLs, we can make them less broken than they are on the KA site. :astonished:

**Comment by jamalex** [comment]
Hmm, actually, looks like the readable id might have changed; no reference to the multi-dash version in the API:
http://www.khanacademy.org/api/v1/videos/uvgBSJPiQ8Y

**Comment by mjptak** [comment]
As the perseus library builds into physics, chemistry and beyond I would expect that hint links to websites and/or pdfs for the hints may become more predominant.  (humankind does not learn by video and exercise alone).   I saw such in some of the computer science and algorithm stuff.   There seems to be a fair bit of this already in the partner content like that coming out of the California Academy of Science(led by UW alum Foley).

**Comment by rtibbles** [comment]
Missing image on this exercise:
![image](https://cloud.githubusercontent.com/assets/1680573/6424897/864eeac0-bec1-11e4-9bbc-c5e2b0601521.png)

**Comment by rtibbles** [comment]
Missing image on:
![image](https://cloud.githubusercontent.com/assets/1680573/6478396/05aeaf8c-c258-11e4-89bb-e95a982ae290.png)

**Comment by rtibbles** [comment]
![image](https://cloud.githubusercontent.com/assets/1680573/6566592/d0b7e12a-c678-11e4-914b-6c96841ecee3.png)

**Comment by aronasorman** [comment]
So we just didn't download it at all... strange. Maybe this is one of the timeouts that we experienced @jamalex?

**Comment by rtibbles** [comment]
Worth running over all the assessment items and just double checking we have the file, and if not, trying to download again?

**Comment by aronasorman** [comment]
I believe this is a blocker for release.

**Comment by aronasorman** [comment]
I'll write up a script to download the remaining missing items.

**Comment by jamalex** [comment]
Everything identified here should be fixed in #3342. With the exception of the towers-of-hanoi exclusion (which will require #3342 to be merged first), the latest assessment items can be tested by force-updating using:
```
bin/kalite manage unpack_assessment_zip --force http://eslgenie.com/media/assessment_item_resources.zip
```

**Comment by rtibbles** [comment]
Fixed in #3342
